recipes/libntv2/all/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
I'll try to find time to do this. Is it a blocker for approving this PR?

Reply:
We try to avoid patching in CCI as may differ from the original behavior from the upstream and because CCI will become the maintainer of a non-official patch. It's documented here: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#policy-about-patching

The `libntv2` already offer a Makefile, much more limited, but with some build support at least. So that's why I asking to send to the upstream first, so in case they accept, we will no longer need to patch it here. Or at least, they can provide some feedback.